### PR TITLE
test: create the integration-cli test binary

### PR DESCRIPTION
**Pull request by sunyuan3**
Signed-off-by: Yuan Sun <yile.sy@alibaba-inc.com>

**1.Describe what this PR did**
Create the integration-cli test binary.  Avoid setting up golang environment in host.

**2.Does this pull request fix one issue?** 
No

**3.Describe how you did it**
Build integration cli test binary in container and execute the binary in host.

**4.Describe how to verify it**
hack/make.sh

**5.Special notes for reviews**
None

### CODE REVIEW

**Comment by allencloud** [comment]
Could you help to review this? @Letty5411

File: hack/make.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ function build_test ()
 
 	# build pouch binaries and execute unit tests.
 	echo "Build pouch binaries and execute unit tests."
-	docker run -ti -v `pwd`:/go/src/github.com/alibaba/pouch pouch:test bash -c "make check && make build && make unit-test"
+	docker run -ti -v `pwd`:/go/src/github.com/alibaba/pouch pouch:test bash -c "make check && make build && make unit-test && cd test && go test -c -o integration-test"
```
Comment (Letty5411) [inline hack/make.sh]: Could we add an entrypoint in Makefile?

Comment (sunyuan3) [reply hack/make.sh]: Yes, I can.  I will create another pull request to add binary, unittest, integration test.

**Comment by allencloud** [comment]
LGTM

File: hack/make.sh (hunk 1 above)
Comment (Letty5411) [reply hack/make.sh]: OK

**Pull request merged**